python/source/org/libreoffice/ide/eclipse/python/PythonProjectHandler.java

```diff
@@ -340,34 +259,4 @@ public String getRegistrationClassName(IUnoidlProject pProject) {
         return folders.toArray(new IFolder[folders.size()]);
     }
 
-    // --------------------------------------------- Jar finding private methods
-
-    /**
-     * returns the path of all the kept jars contained in the folder pointed by path.
-     *
-     * @param pOoo
-     *            the OOo instance from which to get the jars
-     * @return a vector of Path pointing to each jar.
-     */
-    public static Vector<Path> findJarsFromPath(IOOo pOoo) {
-        Vector<Path> jarsPath = new Vector<Path>();
-
-        //        String[] paths = pOoo.getClassesPath();
-        //        for (String path : paths) {
-        //            Path folderPath = new Path(path);
-        //            File programFolder = folderPath.toFile();
-        //
-        //            String[] content = programFolder.list();
-        //            for (int i = 0, length = content.length; i < length; i++) {
-        //                String contenti = content[i];
-        //                if (isKeptJar(contenti)) {
-        //                    Path jariPath = new Path(path + "/" + contenti); //$NON-NLS-1$
-        //                    jarsPath.add(jariPath);
-        //                }
-        //            }
-        //        }
-
-        return jarsPath;
-    }
-
```

Comment:
The code that was not required in the creation/Running/Export of Python Project have been removed.